spec/lib/insert/insert_tickets_spec.rb

```diff
@@ -357,6 +357,13 @@ def success_expectations
           it 'increases TicketPurchase by 1' do
             expect { result }.to change {TicketPurchase.count}.by(1)
           end
+
+          it 'creates a subtransaction payment whose creation date matches the legacy payment\'s date' do
+            legacy_payment = result['payment']
+            subtransaction_payment = legacy_payment.subtransaction_payment
+
+            expect(subtransaction_payment.created).to eq(legacy_payment.created_at)
```

Comment:
This should be the `legacy_payment.date`. On offsite payments, date will not be the same as the `created_at` time.

```suggestion
            expect(subtransaction_payment.created).to eq(legacy_payment.date)
```